editor/src/messages/portfolio/document/node_graph/node_graph_message_handler.rs

```diff
@@ -101,6 +102,8 @@ pub struct NodeGraphMessageHandler {
 	pub selected_nodes: Vec<graph_craft::document::NodeId>,
 	#[serde(skip)]
 	pub widgets: [LayoutGroup; 2],
+	#[serde(skip)]
+	pub is_drawing: bool,
```

Comment:
Based on my read of the code, this is only updated when the Imaginate or Node Graph Frame tools send the update messages-- and other tools don't update it. To avoid confusion, I'd suggest adding a comment to this bool explaining what it does and how it's only used by certain tools that need it like Imaginate and Node Graph Frame.

Reply:
I renamed it to `is_drawing_node_graph_frame`

Reply:
Perfect